Assets/MixedRealityToolkit-UnitTests/Editor/Utilities/Extensions/GameObjectExtensionsTests.cs

```diff
@@ -221,7 +221,7 @@ public void GetFullPathTests(string result, params string[] names)
             }
 
 #pragma warning disable 618
-            Assert.That(parent.GetFullPath(), Is.EqualTo(result));
+            Assert.That(parent.transform.GetFullPath(), Is.EqualTo(result));
```

Comment:
What obsolete code is the 618 suppressing?

Reply:
We can most likely remove the 618

Reply:
Removed